scripts/commands/updateApiDocs.ts

```diff
@@ -272,7 +278,10 @@ async function convertHtmlToMarkdown(
       title: pkg.title,
       name: pkg.name,
       version,
-      changelogUrl: `https://github.com/${pkg.githubSlug}/releases`,
+      releaseNotesUrl:
+        pkg.name !== "qiskit"
+          ? `/api/${pkg.name}/release_notes`
```

Comment:
This is super nitty: can we call it `release-notes`? We've been using `-` for URL and file names in this project.